Tests/Admin/AdminTest.php

```diff
@@ -379,7 +379,7 @@ public function testGetBreadCrumbs()
         $commentAdmin->getBreadcrumbs('reply');
     }
```

Comment:
@core23 : here are the tests. Now that you know that they still pass, I will move them to `BreadcrumbsBuilderTest`